scripts/submit_form.js

```diff
@@ -0,0 +1,113 @@
+// Google Apps Script to submit a form to a Google Sheet and send an email notification
+
+var SCRIPT_PROPERTIES = PropertiesService.getScriptProperties();
+const EMAIL_ADDRESS = "";  // set this!
+
+const formIdToNameMap = new Map([
+  ["feedback", "Give Feedback"],
+  ["report_missing_info", "Report Missing or Outdated Information"],
+  ["partner_with_freefrom", "Partner with FreeFrom"],
+  ["build_collective_survivor_power", "Build Collective Survivor Power"],
+  ["policy_ideas", "Share your Policy Ideas"],
+]);
+
+function setup() {
+  var doc = SpreadsheetApp.getActiveSpreadsheet();
+  SCRIPT_PROPERTIES.setProperty("key", doc.getId());
+}
+
+function doPost(request) {
+  const { _, postData: { contents, type } = {} } = request;
+
+  if (type !== "application/json") {
+    return ContentService.createTextOutput(
+      JSON.stringify({
+        result: "error",
+        error: "expected type application/json",
+      })
+    ).setMimeType(ContentService.MimeType.JSON);
+  }
```

Comment:
See note about error handling below! The only client calling this should really only be our Flask backend, so I don't anticipate this one needing fancy handling